src/Microsoft.Security.Utilities.Core/DataClassification/DAT101_004_Float.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class Float : RegexPattern
+    {
+        public Float()
```

Comment:
>Float [](http://example.com/codeflow?start=15&length=5)

ah! you added float.

your regex looks good, though I'd add a case with an explicit leading zero for the negative case, i.e., `-0.5`

also for this rule, should we attempt any parsing?

note that this rule won't find `12,5` in locales where the comma is the locale-specific decimal separator. someone somewhere must have made a mistake by configuring data in a culture-sensitive locale!

just thinking aloud, we can always ship some basics and come back to the topic.

Reply:
you didn't add my test case of `-0.5` but I am not super worried about it.